README.md

```diff
@@ -0,0 +1,68 @@
+# Awesome-books_JavaScript-objects
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> In this project, we refactor the awesome books project to use ES6 syntax and package management using npm.
+
+## Screenshot
+
+![app-screenshot](./screenshot.png)
+
+## Built With
+
+- HTML/CSS, Javascript
+- Git/Github, CLI, npm
+- Luxon.js
+
+## Live Demo
+
+[Live Demo Link](https://jssol.github.io/awesome-books-es6/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
```

Comment:
- [ ] Let's clean up the documentation. Since this **Prerequisites** section has no description, kindly remove it or add a description.